### PR TITLE
Attribute is required for color index & filter.

### PR DESCRIPTION
**Pull request by epilande**

### CODE REVIEW

File: src/js/components/Aggregate.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ export default class Aggregate extends Component {
 Aggregate.propTypes = {
   a11yTitleId: PropTypes.string,
   a11yDescId: PropTypes.string,
+  attribute: PropTypes.string.isRequired,
```
Comment (ericsoderberghp) [inline src/js/components/Aggregate.js:93]: There is already a "name" property that should be used for this. Why the addition of "attribute" in addition to "name"?

Comment (epilande) [inline src/js/components/Aggregate.js:93]: The `name` property is used for the tile name.

``` javascript
// src/js/reducers/dashboard.js
const initialState = {
  tiles: [
    {
      name: 'Server Profile Changes',
      history: true,
      type: 'area',
      wide: true,
      category: 'server-profiles',
      attribute: 'state',
      interval: 'days',
      count: 8
    },
    {
      name: 'Active Alerts',
      path: '/activity',
      type: 'circle',
      category: 'alerts',
      query: new Query('state:Active'),
      attribute: 'status'
    },
    {
      name: 'Server Profiles',
      type: 'circle',
      category: 'server-profiles',
      attribute: 'status'
    },
    {
      name: 'Server Hardware',
      type: 'distribution',
      category: 'server-hardware',
      attribute: 'model'
    }
  ]
};
```

Comment (ericsoderberghp) [inline src/js/components/Aggregate.js:93]: That's in a Ferret context. Aggregate was changed to be more self consistent, like the changes to Filter. So, Ferret should be changed to map the attribute name to the "name" property.

Comment (epilande) [inline src/js/components/Aggregate.js:93]: Ahh gotcha. Makes sense

**Pull request closed without merging**